fuzz/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "snarkos-fuzz"
+version = "0.0.0"
```

Comment:
Do we want to match the snarkOS version here?

Reply:
this was auto-generated, I don't think it's intended to ever be published; if anything, I think we should rather add some README which would be updated with subsequent runs, stamped with the specific SHA of the repo (as the version is not updated as often)

Reply:
Perfect—so long as we keep track of what code corresponds to which run 👍